pkg/envoy/route/config.go

```diff
@@ -264,7 +264,7 @@ func sanitizeHTTPMethods(allowedMethods []string) []string {
 func NewRouteConfigurationStub(routeConfigName string) *xds_route.RouteConfiguration {
 	routeConfiguration := xds_route.RouteConfiguration{
 		Name:             routeConfigName,
-		ValidateClusters: &wrappers.BoolValue{Value: true},
+		ValidateClusters: &wrappers.BoolValue{Value: false},
```

Comment:
This change change from `true` to `false` seems somehow unrelated to the rest of the channel / pubsub changes.

Is this necessary?

Reply:
It's an unintentional behavior discovered while debugging all these for this patch.

I can make it as a standalone, but anyway I'll have to rebase as event-based behavior works definitely best with this change.  I'll leave the decision up to you. :snowman:

Reply:
Could you drop a comment on why this is explicitly set to false.